### PR TITLE
Use affinity/toleration for services apps in the cluster

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Now we have a new instance group in the cluster to hold the utilities services, this PR sets the affinity/toleration to use those nodes.
In case the nodes does not exist it will be scheduled in the "normal" nodes

Also remove the prometheus since we are not using that anymore in favor of prometheus operator

#### Ticket Link
JIRA: https://mattermost.atlassian.net/browse/MM-30295

#### Release Note

```release-note
Use affinity/toleration for services apps in the cluster
```

### CODE REVIEW

File: helm-charts/prometheus_operator_values.yaml [hunk 1]
```diff
@@ -1841,7 +1857,7 @@ prometheus:
       limits:
         cpu: "3"
         memory: 15Gi
```
Comment (stylianosrigas) [inline helm-charts/prometheus_operator_values.yaml:1859]: @cpanato can we just increase this as well to be able to use the 64GB we will give?

**Review by stylianosrigas — APPROVED** [review]
Just one change regarding limit ;)

**Review by gabrieljackson — APPROVED** [review]
As long as we have run this on an existing cluster to ensure there are no issues with the removal of the prometheus utility then looks good to me!

File: helm-charts/prometheus_operator_values.yaml (hunk 1 above)
Comment (cpanato) [reply helm-charts/prometheus_operator_values.yaml:1859]: will add 60

Comment (cpanato) [reply helm-charts/prometheus_operator_values.yaml]: @stylianosrigas done PTAL

**Pull request merged**